.github/workflows/update-dependencies.yml

```
@@ -0,0 +1,87 @@
+name: Update dependencies
+
+on:
+  # Allow manual executions
+  workflow_dispatch:
+  # Run nightly
+  schedule:
+    - cron: '0 0 * * *'
+
+jobs:
+  propose_github_release_updates:
+    runs-on: ubuntu-22.04
+    strategy:
+      matrix:
+        include:
+          - key: kolla
+            path: src/kayobe-config/etc/kayobe/stackhpc.yml
+            repository: stackhpc/kolla
+            search_regex: 'stackhpc_kolla_source_version\:.*$'
+            prefix: 'stackhpc_kolla_source_version\: '
+
+          - key: kolla-ansible
+            path: src/kayobe-config/etc/kayobe/stackhpc.yml
+            repository: stackhpc/kolla-ansible
+            search_regex: 'stackhpc_kolla_ansible_source_version\:.*$'
+            prefix: 'stackhpc_kolla_ansible_source_version\: '
+
+          - key: kayobe
+            path: src/kayobe-config/requirements.txt
+            repository: stackhpc/kayobe
+            search_regex: '@stackhpc\/.*$'
```

I could see this causing a problem if we add more requirements to this file. Could we include `kayobe` in the search regex?
e.g.
```suggestion
            search_regex: 'kayobe@stackhpc\/.*$'
```